src/main/java/pl/szymonleyk/spotifypartyapp/spotify/api/client/response/DeviceResponse.java

```diff
@@ -0,0 +1,17 @@
+package pl.szymonleyk.spotifypartyapp.spotify.api.client.response;
+
+import lombok.*;
+
+import java.io.Serializable;
+
+@Getter
+@Setter
+public class DeviceResponse implements Serializable {
+    private String id;
+    private boolean is_active;
```

Comment:
Pozamieniaj nazwy pól ze snake case, na camel case :)

czyli trzymaj się Javowej konwencji.

Tutaj dałeś snake case, ponieważ w api tak są nazwane pola, masz rację, ale załątwiłem to w inny sposób, przez co mam i jedno i drugie.

Czyli. pola nazywamy po "naszemu" tak jak chcemy mieć w Javie. Natomiast mapowanie robi się jako snake case, dzięki ustawionemu parametrowi w application.properties
spring.jackson.property-naming-strategy=SNAKE_CASE

Dlatego śmiało możesz pozamieniać te pola :)